h5p-bildetema/src/hooks/useSiteLanguage.ts

```diff
@@ -1,5 +1,31 @@
+import { Language } from "common/types/types";
+import { useNewDBContext } from "common/hooks/useNewDBContext";
+import { attributeLanguages } from "common/constants/languages";
 import { useL10n } from "./useL10n";
 
-export const useSiteLanguage = (): string => {
+export const useSiteLanguageString = (): string => {
   return useL10n("htmlLanguageCode");
 };
+
+export const useSiteLanguage = (): Language => {
+  const { languages } = useNewDBContext();
+  const siteLanguageString = useSiteLanguageString();
+
+  const siteLanguage = languages.find(
+    lang =>
+      siteLanguageString ===
+      (attributeLanguages as Record<string, string>)[lang.code],
+  );
+
+  if (!siteLanguage) {
+    return { code: "nob", label: "Bokmål", rtl: false } as Language;
+  }
+
+  return siteLanguage;
+};
+
+// Without react router
+export const getSiteLanguagePath = (): string => {
```

Comment:
Add   if (typeof window === "undefined") return "";  to prevent an SSR error

Reply:
Added!